src/convertToObject.js

```diff
@@ -6,7 +6,23 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const trimmedString = sourceString
+    .split(';')
+    .filter((item) => item.trim() !== '');
+
+  const propertyValuePairs = trimmedString.map((item) => {
+    const [property, value] = item.split(':');
+
+    return [property.trim(), value.trim()];
+  });
+
+  const resultingObject = {};
+
+  propertyValuePairs.map((item) => {
```

Comment:
```suggestion
  propertyValuePairs.map(([key, value]) => {
```
Use array destructuring